AppAuth-JS/built/authorization_service_configuration_test.js

```diff
@@ -0,0 +1,73 @@
+"use strict";
```

Comment:
This is generated source file. All the files in built/ folder are generated JS file for the Google's lib fork